examples/tutorials/tutorial_2dcrm_example.py

```diff
@@ -0,0 +1,211 @@
+                    geometry['symmetry']
+                    ]
+                )
+            ]
+        ),
+        reference_geometry=ReferenceGeometry(
+            moment_center=[0.25,0.005,0],
+            moment_length=[1,1,1],
+            area=0.01
+        ),
+        operating_condition=create_operating_condition_from_mach_reynolds(
+            mach=0.2,
+            reynolds=5e+6,
+            temperature=272.1,
+            alpha=16*u.deg,
+            beta=0*u.deg
+        ),
+        time_stepping=Steady(
+            max_steps=3000,
+            CFL=RampCFL(
+                initial=20,
+                final=300,
+                ramp_steps=500
+            )
+        ),
+        outputs=[
+            VolumeOutput(
+                name='VolumeOutput',
+                output_fields=[
+                    'primitiveVars',
+                    'vorticity',
+                    'residualNavierStokes',
+                    'residualTurbulence',
+                    'Cp',
+                    'Mach',
+                    'qcriterion',
+                    'mut'
+                    ]
+            ),
+            SurfaceOutput(
+                name='SurfaceOutput',
+                surfaces=geometry["*"],
+                output_fields=[
+                    'primitiveVars',
+                    'Cp',
+                    'Cf',
+                    'CfVec',
+                    'yPlus'
+                ]
+            )
+        ],
+        models=[
+            Wall(
+                surfaces=[
+                    geometry['wing'],
+                    geometry['flap'],
+                    geometry['slat'],
+                    geometry['wingTrailing'],
+                    geometry['flapTrailing'],
+                    geometry['slatTrailing']
+                ],
+                name='wall'
+            ),
+            Freestream(
+                surfaces=farfield.farfield,
+                name='Freestream'
+            ), 
+            SlipWall(
+                surfaces=farfield.symmetry_planes,
+                name='slipwall'
+            ),
+            Fluid(
+                navier_stokes_solver=NavierStokesSolver(
+                    absolute_tolerance=1e-11,
+                    relative_tolerance=1e-2,
+                    linear_solver=LinearSolver(max_iterations=35),
+                    kappa_MUSCL=0.33,
+                    order_of_accuracy=2,
+                    update_jacobian_frequency=4,
+                    equation_evaluation_frequency=1
+                ), 
+                turbulence_model_solver=SpalartAllmaras(
+                    absolute_tolerance=1e-10,
+                    relative_tolerance=1e-2,
+                    linear_solver=LinearSolver(max_iterations=25),
+                    order_of_accuracy=2,
+                    update_jacobian_frequency=4,
+                    equation_evaluation_frequency=1
+                )
```

Comment:
default order of accuracy = 2
default update_jacobian_frequency = 4
default eqation_evaluation_frequency = 4 instead of 1
linear_solver while it is a default setting, the max iterations parameter by default is 20, whereas the case states 25
I stated them explicitly because they were stated in the case json but I can remove the parameters that overlap with default values

while relative_tolerance is not used in steady state it was still defined in the case.json here:
"turbulenceModelSolver": {
        "modelType": "SpalartAllmaras",
        "absoluteTolerance": 1e-10,
        "relativeTolerance": 1e-2,
        "linearIterations": 25,
        "kappaMUSCL": -1,
        "orderOfAccuracy": 2,
        "updateJacobianFrequency": 4,
        "equationEvalFrequency": 1

Reply:
I deleted the inputs you mentioned other than equation_evaluation_frequency=1